version/version.go

```diff
@@ -87,3 +94,67 @@ func Handler() http.Handler {
 		enc.Encode(v)
 	})
 }
+
+// versionMultiplier consts are used to create a comparable and reversible integer version from a semver string
+// by using 1 million, 1 thousand, and 1 for each part we avoid collisions as long as all parts are less than 1 thousand
+const (
+	majorVersionMultiplier int = 1000000
+	minorVersionMultiplier int = 1000
+	patchVersionMultiplier int = 1
```

Comment:
If we wanted to support a possible future date based form, do these need to have an additional zero?

Counterpoint, maybe not, so that any future 2024, is _bigger_ than a potential 3.0

Reply:
yeah i like the counterpoint here!